scripts/dev/templates/agent/Dockerfile.builder

```diff
@@ -0,0 +1,18 @@
+FROM curlimages/curl:7.76.1 as builder
```

Comment:
I think this one can be replaced by a scratch image like

```Dockerfile
FROM scratch

ADD https://mciuploads.s3.amazonaws.com/mms-automation/mongodb-mms-build-agent/builds/automation-agent/prod/mongodb-mms-automation-agent-${agent_version}.${agent_distro}.tar.gz /data/mongodb-agent.tgz
ADD https://downloads.mongodb.org/tools/db/mongodb-database-tools-${tools_distro}-${tools_version}.tgz /data/mongodb-tools.tgz

ADD agent/LICENSE /data/licenses
```

Instead of the multi-stage build with curl.

Reply:
ah much better!